tests/xtro-sharpie/Helpers.cs

```diff
@@ -66,6 +66,7 @@ public static partial class Helpers {
 			{ "NFCVASErrorCode", "VasErrorCode" },
 			{ "NFCVASMode", "VasMode" },
 			{ "NFCISO15693RequestFlag", "RequestFlag" },
+			{ "CGImagePropertyTGACompression", "CGImagePropertyTgaCompression"}
```

Comment:
was it needed ? I thought it was made to ignore casing a while ago...

Reply:
TBH I am not sure :P I just added it before running xtro, I can tests and remove if you want

Reply:
#YOLO'ed in the web editor